upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1400,37 +1359,6 @@ else
   echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
-state_name="VALIDATE_SPIRE_POSTGRES_HEALTH"
```

Comment:
No need to wait for spire postgres. We restart it and wait for it to come online in FIX_POSTGRES.